docs/building_on_linux.md

```diff
@@ -33,11 +33,21 @@ schismtracker directory and running:
 To build Schism Tracker, you should set up a build-directory. From the
 schismtracker directory:
 
-    mkdir -p build && cd build && ../configure && make
+    mkdir -p build && cd build && ../configure && \
+    make -j $(nproc || sysctl -n hw.ncpu || echo 2)
 
 The resulting binary `schismtracker` is completely self-contained and can be
 copied anywhere you like on the filesystem.
 
+You can specify custom compiler flags, e.g. to optimize schismtracker
+stronly, system-dependently:
+
+    make clean  # recompiling needed after changing compiler setting
+    ../configure --enable-extra-opt
+    make -j $(nproc || sysctl -n hw.ncpu || echo 2)
+
+For debugging, and other settings, see ```../configure --help```.
```

Comment:
Only multi-line code snippets require three back-ticks. See how `schismtracker` is formatted on line 39.